playbooks/common/openshift-cluster/upgrades/v3_0_to_v3_1/upgrade.yml

```diff
@@ -517,24 +517,29 @@
     - _default_router.rc == 0
     - "'false' in _scc.stdout"
     command: >
-      {{ oc_cmd }} patch scc/privileged -p '{"allowHostPorts":true,"allowHostNetwork":true}' --loglevel=9
+      {{ oc_cmd }} patch scc/privileged -p
+      '{"allowHostPorts":true,"allowHostNetwork":true}' --loglevel=9
```

Comment:
Should we drop the loglevel option while we're here, probably an artifact from debugging when this was originally written?